python/packages/azurefunctions/agent_framework_azurefunctions/_entities.py

```diff
@@ -181,12 +245,31 @@ async def run_agent(
             logger.error(f"Error type: {type(exc).__name__}")
             logger.error(f"Full traceback:\n{error_traceback}")
 
+            # Create error response and store it in conversation history so polling can find it
+            from agent_framework import ChatMessage, ErrorContent
+
+            # Create error message
```

Comment:
NIT: Maybe create a helper for this to keep the changes localized to `durable_agent_state.py` file?